### DISCUSSION TITLE
http3 requires udp

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2825] [Questions] [answered]
Caddy allows http3 and the readme also mentions it, but in the container.json, udp is not exposed, this will cause http3 to not work

**Comment by szaimen** [comment]
I have simply not tested http3 yet 😅

So thanks for the heads-up!

I guess we'll need to adjust the docs then as well to also open port 443/udp in the router/firewall.

**Comment by szaimen** [comment, marked as answer]
PR in https://github.com/nextcloud/all-in-one/pull/2828

**Reply by szaimen** [reply]
This is now released with v6.3.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
